### PR TITLE
Add Ekinops 360 support

### PR DESCRIPTION
**Pull request by Najihel**
Add basic Ekinops 360 support :

GitHub Issue : https://github.com/librenms/librenms/issues/9451
LibreNMS Community : https://community.librenms.org/t/new-device-ekinops/11640

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/new-device-ekinops/11640/2

**Comment by Najihel** [comment]
FYI :

I can't find a way to force the hostname via the inc.php file.

The hostname need to be overwrite because SysName isn't customizable on Ekinops product. It's always "ekinops product" but the real hostname is stored on a specific OID.

Maybe there is a way to do that, but I didn't find it yet.

**Comment by Jellyfrog** [comment]
Missing json test files

File: includes/polling/os/ekinops-360.inc.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (Jellyfrog) [inline includes/polling/os/ekinops-360.inc.php:1]: This should be converted to the new format, see for example https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php#L34

Comment (Najihel) [reply includes/polling/os/ekinops-360.inc.php:1]: I haven't access to Ekinops product anymore. Maybe another contributor can take over for this basic support.

**Comment by Najihel** [comment]
@h-barnhart got a head start on my work : https://github.com/h-barnhart/librenms

**Pull request closed without merging**